### DISCUSSION TITLE
Diagramas en Practica de Requerimientos

### DISCUSSION THREAD

**Discussion by LuisFerVR** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#31] [Q&A]
¿Se tendrán que realizar los diagramas que solicita el documento?  Son muchos y por tiempo, no es tan producente. por ello mi pregunta. Gracias!

**Comment by AngelCruzO** [comment]
Puedes buscar una plantilla mas simple, en mi caso solo hice requisitos funcionales y no funcionales

**Reply by LuisFerVR** [reply]
Gracias por la sugerencia!

**Reply by AngelCruzO** [reply]
De nada.
